clientv3/integration/client_test.go

```diff
@@ -61,3 +63,104 @@ func TestKVPut(t *testing.T) {
 		}
 	}
 }
+
+func TestKVRange(t *testing.T) {
+	defer testutil.AfterTest(t)
+
+	clus := integration.NewClusterV3(t, &integration.ClusterConfig{Size: 3})
+	defer clus.Terminate(t)
+
+	kv := clientv3.NewKV(clus.RandClient())
+
+	keySet := []string{"a", "b", "c", "c", "c", "foo", "foo/abc", "fop"}
+	for i, key := range keySet {
+		if _, err := kv.Put(key, "", lease.NoLease); err != nil {
+			t.Fatalf("#%d: couldn't put %q (%v)", i, key, err)
+		}
+	}
+
+	tests := []struct {
+		begin, end string
+		rev        int64
+
```

Comment:
I put arguments that are needed for `kv.Range` and compare the returned `storage.KeyValue`. Any suggestion?